tests/escapedHashtags.js

```diff
@@ -58,33 +69,32 @@ define([
                     ["#form/text1", "`#form/text1`"],
                     ["/data/text1", "`#form/text1`"],
                     ["`#form/text1`", "`#form/text1`"],
+
+                    ["#form/text1 -1", "`#form/text1` - 1"],
+                    ["/data/text1 -1", "`#form/text1` - 1"],
+                    // ideally no change, but too hard right now (to much extra parsing going on)
```

Comment:
Yes. We really only need to escape hashtags when we have an invalid xpath expression. Otherwise escaping is not necessary because the hashtags can be found with normal parsing. Using escaped hashtags all the time might be a bit simpler in vellum, although the escaping logic needs to be used in more places so this is debatable. However, it makes it harder for end-users to hand-edit XML and copy/pasted questions in Excel.